db/dataset.py

```diff
@@ -315,3 +355,147 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def _get_classid_for_dataset(connection, dataset_id, class_name):
+    query = sqlalchemy.text("""
+      SELECT id
+        FROM dataset_class
+       WHERE name = :name
+         AND dataset = :dataset_id""")
+    result = connection.execute(query, {"name": class_name, "dataset_id": dataset_id})
+    if result.rowcount < 1:
+        raise exceptions.NoDataFoundException("No such class exists.")
+    clsid = result.fetchone()
+    return clsid[0]
+
+
+def add_recordings(dataset_id, class_name, recordings):
+    """Adds new recordings to a class in a dataset.
+
+    If any given recording already exist in this class it is not added again.
+
+    Args:
+        dataset_id: the uuid of the dataset
+        class_name: The class to add the recordings to
+        recordings: the recordings to add
```

Comment:
"List of recordings (MBID strings) to add" might be a slightly better description.